djstripe/models.py

```diff
@@ -2155,6 +2170,10 @@ def plan(self):
 
 
 class UpcomingInvoice(Invoice):
+
+    class Meta:
+        abstract = djstripe_settings.ABSTRACT_MODELS
```

Comment:
Leave it off UpcomingInvoice. It'll get inherited anyway.

Reply:
@jleclanche nope. Meta classes are not inherited. If I remove Meta it'll become regular model